src/ensembl/src/analyticsHelper.ts

```diff
@@ -6,6 +6,20 @@ export type AnalyticsOptions = {
   value?: number;
 };
 
+/*
+  These dimensions are defined here:
+  Analytics -> 2020 -> Admin -> Custom Definitions -> Custom Dimensions
+*/
+export enum CustomDimensions {
+  SPECIES = 'dimension1',
+  PAGEVIEW = 'dimension2'
+}
+
+export type CustomDimensionOptions = {
+  diemension: CustomDimensions;
```

Comment:
Here and throughout: `diemension` -> `dimension`

Reply:
Remove this definition as it's not needed anymore.